Libplanet/PoS/UnbondingSet.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using Bencodex.Types;
+using Libplanet.Action;
+
+namespace Libplanet.PoS
+{
+    internal class UnbondingSet : UnbondingSetInfo
```

Comment:
How about making static this class and method? This class and methods do not have captured state.

Reply:
For example, below method removes item from RedelegationAddressSet.
```c#
        internal IAccountStateDelta DelRedelegationAddressSet(
            IAccountStateDelta states, Address redelegationEntryAddress)
        {
            RedelegationAddressSet.Remove(redelegationEntryAddress);
            states = states.SetState(Address, Serialize());
            return states;
        }
```

Reply:
Oh, yeah, my bad. I mean, if `RedelegateionAddressSet` makes a method property too, It can be a static method.

Reply:
> if RedelegateionAddressSet makes a method property too, It can be a static method.
I think I'm not understood. Could you explain a bit more?